catalogs/bundled-catalog/src/main/java/com/datastrato/gravitino/catalog/common/ClassProvider.java

```diff
@@ -36,6 +36,7 @@
  * Therefore, this module is used to store the required classes for the query engine's
  * functionality.
  */
+@SuppressWarnings("UnusedVariable")
```

Comment:
Could you explain why we add this annotation?

Reply:
cc @yuqi1129 Could you give some advice?

Reply:
It's a class that holder class that may be needed by query engines like `Spark` and `Trino` and won't be used directly in the project.

<img width="831" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/7bd1cd97-69d4-418e-812a-f3d75cbfe74a">